[2025-08-29T03:32:51.575+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.transform manual__2025-08-29T03:32:42.729072+00:00 [queued]>
[2025-08-29T03:32:51.590+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.transform manual__2025-08-29T03:32:42.729072+00:00 [queued]>
[2025-08-29T03:32:51.591+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-08-29T03:32:51.609+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform> on 2025-08-29 03:32:42.729072+00:00
[2025-08-29T03:32:51.630+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'transform', 'manual__2025-08-29T03:32:42.729072+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp73w4bc2m']
[2025-08-29T03:32:51.635+0000] {standard_task_runner.py:88} INFO - Job 39: Subtask transform
[2025-08-29T03:32:51.625+0000] {standard_task_runner.py:60} INFO - Started process 1379 to run task
[2025-08-29T03:32:51.673+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-29T03:32:51.732+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.transform manual__2025-08-29T03:32:42.729072+00:00 [running]> on host 45d0ac61cfd5
[2025-08-29T03:32:51.896+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-08-29T03:32:42.729072+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-29T03:32:42.729072+00:00'
[2025-08-29T03:32:51.899+0000] {logging_mixin.py:188} INFO - Transforming (fake)...
[2025-08-29T03:32:53.902+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-29T03:32:53.949+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=transform, execution_date=20250829T033242, start_date=20250829T033251, end_date=20250829T033253
[2025-08-29T03:32:54.030+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-29T03:32:54.137+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
